demisto_sdk/commands/content_graph/common.py

```diff
@@ -276,6 +276,81 @@ def by_schema(path: Path, git_sha: Optional[str] = None) -> "ContentType":
+        if self == ContentType.PREPROCESS_RULE:
+            return "PreProcess Rules"
+        elif self == ContentType.TRIGGER:
+            return "Triggers Recommendations"  # https://github.com/demisto/etc/issues/48153#issuecomment-1111988526
+        elif self == ContentType.XSIAM_REPORT:
+            return "XSIAM Reports"
+        elif self == ContentType.XDRC_TEMPLATE:
+            return "XDRC Templates"
+        elif self == ContentType.XSIAM_DASHBOARD:
+            return "XSIAM Dashboards"
+        elif self == ContentType.INCIDENT_FIELD:
+            return "Incident Fields"
+        elif self == ContentType.INCIDENT_TYPE:
+            return "Incident Types"
+        elif self == ContentType.INDICATOR_FIELD:
+            return "Indicator Fields"
+        elif self == ContentType.INDICATOR_TYPE:
+            return "Indicator Types"
+        elif self == ContentType.PREPROCESS_RULE:
+            return "PreProcess Rules"
+        elif self == ContentType.GENERIC_TYPE:
+            return "Object Types"
+        elif self == ContentType.GENERIC_FIELD:
+            return "Object Fields"
+        elif self == ContentType.GENERIC_DEFINITION:
+            return "Objects"
+        elif self == ContentType.GENERIC_MODULE:
+            return "Modules"
+        elif self == ContentType.PARSING_RULE:
+            return "Parsing Rules"
+        elif self == ContentType.MODELING_RULE:
+            return "Modeling Rules"
+        elif self == ContentType.CORRELATION_RULE:
+            return "Correlation Rules"
+        elif self == ContentType.LAYOUT_RULE:
+            return "Layout Rules"
+        elif self == ContentType.ASSETS_MODELING_RULE:
+            return "Assets Modeling Rules"
+        elif self == ContentType.CASE_LAYOUT_RULE:
+            return "Case Layout Rules"
+        elif self == ContentType.CASE_FIELD:
+            return "Case Fields"
+        elif self == ContentType.CASE_LAYOUT:
+            return "Case Layouts"
+        return f"{self.value}s"
+
+    @staticmethod
+    def convert_header_to_content_type(header: str) -> "ContentType":
+        """
+        Convert Release note header to ContentType.
+        """
+        if header == "Triggers Recommendations":
+            return ContentType.TRIGGER
+        elif header == "Preprocess Rules":
+            return ContentType.PREPROCESS_RULE
+        elif header == "Mappers":
+            return ContentType.MAPPER
+        elif header == "Objects":
+            return ContentType.GENERIC_DEFINITION
+        elif header == "Modules":
+            return ContentType.GENERIC_MODULE
+        elif header == "Object Types":
+            return ContentType.GENERIC_TYPE
+        elif header == "Object Fields":
+            return ContentType.GENERIC_FIELD
+        normalized_header = header.rstrip("s").replace(" ", "_").upper()
+        if not ContentType.__dict__.get(normalized_header):
+            raise ValueError
+        return ContentType[normalized_header]
```

Comment:
equivalent:
```suggestion
        return ContentType(normalized_header)
```

Reply:
ContentType[normalized_header] retrieves an enum member by its name, and ContentType(normalized_header)- Retrieves an enum member by its value, and here I need to return the value by its name so ContentType(normalized_header) doesn't work the same here